.travis.yml

```diff
@@ -396,3 +396,15 @@ after_success:
         echo "deploy to maven snapshot repository is finished";
     fi
     sleep 5s
+  - |
+    fail=0
+      if [ -z "$(ls -A .ci-temp)" ]; then
```

Comment:
fix indentation, it should be same as `fail`

Reply:
Fixed.